hosted_apps/clock/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Why is this needed? Shouldn't we be able to figure out the origin from the start_url? Web apps can't have a separate installOrigin to origin.